### PR TITLE
Limit metadata inserion to the first result

### PR DESCRIPTION
**Pull request by dru-karimcossutti**
replacement was matching \<head> as well as \<header> limit should fix the issue since <head> will always come before anything related to the document.

### CODE REVIEW

File: src/ResponseMerger/NuxtResponseMerger.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ public function mergeResponses(ResponseInterface $backendResponse, ResponseInter
         }
       }
       if ($metatags_html != '') {
-        $page = preg_replace("/<head(.*?)>/", '<head$1>' . $metatags_html, $page);
+        $page = preg_replace("/<head(.*?)>/", '<head$1>' . $metatags_html, $page, 1);
```
Comment (fago) [inline src/ResponseMerger/NuxtResponseMerger.php]: Why can the head tag appear more often and why do we need to treat the first one only? This deservers a comment.

**Review by fago — CHANGES_REQUESTED** [review]

File: src/ResponseMerger/NuxtResponseMerger.php (hunk 1 above)
Comment (dru-karimcossutti) [reply src/ResponseMerger/NuxtResponseMerger.php]: :/ html tags got removed from previous comment,
the issue was that this regex matches `head` as well as `header` tags, header tags are in the body, and metadata would be added multiple times, the limit will solve this as `head` always comes before body.

Comment (fago) [reply src/ResponseMerger/NuxtResponseMerger.php]: ok, got it. That's a good find - but as this is not obvious from the code, the more important is the comment. As said:

>This deservers a comment.

Obviously and as always: in code.

Comment (dru-karimcossutti) [reply src/ResponseMerger/NuxtResponseMerger.php]: Documented in code

**Review by fago — APPROVED** [review]

**Pull request merged**